src/Symfony/Component/VarDumper/Server/Connection.php

```diff
@@ -85,6 +85,7 @@ public function write(Data $data): bool
     private static function nullErrorHandler(int $t, string $m)
     {
         // no-op
+        return null;
```

Comment:
Error handlers should return `bool`.

```suggestion
        return true;
```

Reply:
returning nothing is also fine, so I added `void` instead